xcdat/regridder/accessor.py

```diff
@@ -185,7 +197,7 @@ def horizontal(
         self,
         data_var: str,
         output_grid: xr.Dataset,
-        tool: RegridTool = "xesmf",
+        tool: RegridTool,
```

Comment:
Just an FYI @xCDAT/core-developers: `"xesmf"` is no longer the default when calling `ds.regridder.horizontal()`. The reason I changed this is because `xesmf` is now an optional dependency, and it might confuse users to receive a `ModuleNotFoundError` for this method if they don't have "xesmf" installed (and/or realize "regrid2" is also an option).

The user should explicitly decide which tool to use for horizontal regridding.

Reply:
I think we should leave this as-is. I think the vast majority of users can and will want to use xesmf.

Reply:
Thanks for the input. Got it, I will revert based on that reasoning.

Reply:
One vote to leave this as-is..

Reply:
> One vote to leave this as-is..

Do you mean "as it was" or should the default be removed?

Reply:
I think Jill agreed with you since you said as-is as well (to keep "xesmf") haha.